Sources/Utilities/KMZendeskChatHandler.swift

```diff
@@ -233,5 +319,56 @@ public class KMZendeskChatHandler {
         }
         return true
     }
+    
+    func updateLastMessageCreatedTime() {
+        let channelKey = NSNumber(value: Int(groupId) ?? 0)
+        guard channelKey != 0 else {
+            print("Failed to fetch messages for group Id")
```

Comment:
It says for group ID, but there's no ID passed. Also, in future we'll have to use a logger that contains prefix, error/info etc. Otherwise, the customer won't be able to easily differentiate b/w their logs and ours.

Reply:
group id already set by updateGroupI function. updated the log. Also we have filed a ticket to optimise the logs , we will handle this prefix feature there